shared/modules/ssgcommon.py

```diff
@@ -249,7 +249,7 @@ def bool_constructor(self, node):
         return self.construct_scalar(node)
 
     # Don't follow python bool case
-    yaml.Loader.add_constructor(u'tag:yaml.org,2002:bool', bool_constructor)
+    yaml.SafeLoader.add_constructor(u'tag:yaml.org,2002:bool', bool_constructor)
```

Comment:
wait... every time we load a yaml we add a constructor to the static class?

Reply:
shouldn't this be called right after we import "yaml" module?

Reply:
@mpreisler it doesn't have to. I moved it to the top after the imports.